test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTrimTest.java

```diff
@@ -59,32 +59,6 @@ public void testCheckpointTrim() throws Exception {
                 .containsKeys("a", "b", "c");
     }
 
-    @Test
```

Comment:
Shouldn't we keep this test to show that this will not cause the trim to fail, and attempt to access the actual address to verify it was trimmed?

Reply:
Done.